helion/autotuner/default_pattern_search.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+import math
+from typing import TYPE_CHECKING
+
+from .. import exc
+from .base_search import FlatConfig
+from .base_search import PopulationMember
+from .base_search import performance
+from .effort_profile import PATTERN_SEARCH_DEFAULTS
+from .pattern_search import PatternSearch
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator
+    from collections.abc import Sequence
+
+    from ..runtime.config import Config
+    from ..runtime.kernel import BoundKernel
+
+
+class DefaultPatternSearch(PatternSearch):
```

Comment:
```suggestion
class QuickPatternSearch(PatternSearch):
```

I worry people won't understand `Default` means the default config.